adhocracy-plus/assets/js/init-picker.js

```diff
@@ -1,15 +1,77 @@
-const $ = require('jquery')
-const datePickerController = require('datepicker')
-const django = require('django')
+import django from 'django'
+import flatpickr from 'flatpickr'
 
-$(function () {
-  const $inputs = $('.datepicker')
+function link (f0, f1) {
+  const date = f0.selectedDates[0]
+  if (date) {
+    // if p0 has a date set that date as minDate for p1
+    f1.set('minDate', f0.formatDate(date, f0.config.dateFormat))
+  }
+  f0.config.onChange.push((selectedDates, dateStr) => {
+    // if date of p0 is changed adapt minDate for p1
+    f1.set('minDate', dateStr)
+  })
+}
+
+function linkDatePickers (flatpickrs) {
+  // normal flatpickers
+  const idStart = 'id_start_date_date'
+  const idEnd = 'id_end_date_date'
+  // (multi-)phase flatpickrs
+  const phaseIds = [
+    'id_phase_set-0-start_date_date',
+    'id_phase_set-0-end_date_date',
+    'id_phase_set-1-start_date_date',
+    'id_phase_set-1-end_date_date',
+    'id_phase_set-2-start_date_date',
+    'id_phase_set-2-end_date_date'
+  ]
+
+  // link non-phase datepickers if exist
+  const fStart = flatpickrs.get(idStart)
+  const fEnd = flatpickrs.get(idEnd)
+  if (fStart && fEnd) {
+    link(fStart, fEnd)
+  }
+
+  // link phase datepickers if exist
+  for (let i = 0; i < phaseIds.length - 1; i++) {
```

Comment:
I think that's on purpose (but can't recall). We always link phase[i] with phase[i+1] (see line 43). If `i` went up to the last element it would try to link it with last element + 1 which doesn't exist